lib/cylc/suite_db_mgr.py

```diff
@@ -290,12 +290,18 @@ def put_task_event_timers(self, task_events_mgr):
             self.db_deletes_map[self.TABLE_TASK_ACTION_TIMERS].append({})
             for key, timer in task_events_mgr.event_timers.items():
                 key1, point, name, submit_num = key
+                # convert namedtuple timer.ctx for JSON serialisation
```

Comment:
Duplicate block to (new) 337:343 - is it worth creating an internal use method & calling it twice?
```
def _conversion_for_JSON(self, tuple):
   """ Convert named tuple timer.ctx for JSON serialisation. """
   if tuple is not None:
      return [type(timer.ctx).__name__, timer.ctx.__getnewargs__()]
```